src/bootstrap/bin/rustdoc.rs

```diff
@@ -44,7 +45,9 @@ fn main() {
         cmd.arg("-Z").arg("force-unstable-if-unmarked");
     }
     if let Some(linker) = env::var_os("RUSTC_TARGET_LINKER") {
-        cmd.arg("--linker").arg(linker).arg("-Z").arg("unstable-options");
+        let mut arg = OsString::from("-Clinker=");
+        arg.push(&linker);
+        cmd.arg(arg);
```

Comment:
The better way to do this is just `.arg("-Clinker").arg(&linker)`

Reply:
tired to do like that but did not get it to work 
when I test with `rustc -Clinker /usr/bin/cc`  I get an error
`error: codegen option `linker` requires a path (C linker=<value>)`

Reply:
Hm, that seems buggy. Oh well, let's leave it like this then.